google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java

```diff
@@ -1177,6 +1178,68 @@ public void testCreateTableWithConstraints() {
     bigquery.delete(tableId);
   }
 
+  @Test
+  public void testCreateTableWithDefaultValueExpression() {
+    String tableName = "test_create_table_with_default_value_expression";
+    TableId tableId = TableId.of(DATASET, tableName);
+    Field stringFieldWithDefaultValueExpression =
+        Field.newBuilder("stringFieldWithDefaultValueExpression", StandardSQLTypeName.STRING)
+            .setMode(Field.Mode.NULLABLE)
+            .setDescription("String field with default value expression")
+            .setDefaultValueExpression("'FOO'")
+            .setMaxLength(150L)
+            .build();
+    Field timestampFieldWithDefaultValueExpression =
+        Field.newBuilder("timestampFieldWithDefaultValueExpression", StandardSQLTypeName.TIMESTAMP)
+            .setMode(Field.Mode.NULLABLE)
+            .setDescription("Timestamp field with default value expression")
+            .setDefaultValueExpression("CURRENT_TIMESTAMP")
+            .build();
+    Schema schema =
+        Schema.of(stringFieldWithDefaultValueExpression, timestampFieldWithDefaultValueExpression);
+    StandardTableDefinition tableDefinition =
+        StandardTableDefinition.newBuilder().setSchema(schema).build();
+
+    // Create table with fields that have default value expression
+    Table createdTable = bigquery.create(TableInfo.of(tableId, tableDefinition));
+    assertNotNull(createdTable);
+
+    // Fetch the created table and its metadata
+    // to verify default value expression is assigned to fields
+    Table remoteTable = bigquery.getTable(DATASET, tableName);
+    Schema remoteSchema = remoteTable.<StandardTableDefinition>getDefinition().getSchema();
+    assertEquals(schema, remoteSchema);
+    FieldList fieldList = remoteSchema.getFields();
+    for (Field field : fieldList) {
+      if (field.getName().equals("timestampFieldWithDefaultValueExpression")) {
+        assertEquals("CURRENT_TIMESTAMP", field.getDefaultValueExpression());
+      }
+      if (field.getName().equals("stringFieldWithDefaultValueExpression")) {
+        assertEquals("'FOO'", field.getDefaultValueExpression());
+      }
+    }
+
+    // Insert value into the created table
+    // to verify default values are inserted when value is missing
```

Comment:
Moot since you've already implemented, but you could have also done this with an INSERT statement.  Thanks for checking the round trip behavior.